### ISSUE TITLE
ScrollBehavior definition is missing 'instant' for window.scrollTo

### ISSUE THREAD

**Issue by RandScullard** [microsoft/TypeScript#47441]
# Bug Report

### ?? Search Terms

instant
ScrollBehavior
window
scrollTo
type definition

### ?? Version & Regression Information

v4.5.4, but it's also mentioned in #46654 (v4.4.4) and #28755 (v3.2.1)

### ? Playground Link

[Playground link with relevant code](https://www.typescriptlang.org/play?#code/O4SwdgJg9sB0DOBjATlANmgKlAFAbwFgAoAAhIBcoAHALhIAYAaYstAUwDNy6mWSAjNgAsAhgDcQUZHQDk4eORFhyM4gF8AlEA)

### ?? Code

```ts
window.scrollTo({
  top: 0,
  left: 0,
  behavior: 'instant' // This should be allowed
})
```

### ?? Actual behavior

The value of `instant` cannot be assigned to `behavior` when using `window.scrollTo`.

### ?? Expected behavior

The value of `instant` is allowed when using `window.scrollTo` according to https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo as well as in real-world testing in major browsers. I just tested `window.scrollTo` with `instant` on Chrome, Firefox, and macOS Safari and it does **not** give the same behavior as `auto`. In my tests, `auto` produced smooth scrolling and `instant` scrolled instantly.

Please note that the MDN documentation specifies different behavior options for `element.scrollTo` and `element.scrollIntoView` vs. `window.scrollTo`. This bug report is specific to `window.scrollTo`. The existing ScrollBehavior type definition will most likely need to be bifurcated to handle all functions correctly.

Until the type definition is fixed, you can get the instant scrolling behavior by casting as any:

```ts
window.scrollTo({ top: 0, left: 0, behavior: 'instant' } as any)
```

**Comment by RandScullard** [comment]
Please note that this issue is very similar to #46654, but that issue was related to `element.scrollTo` and this issue is related to `window.scrollTo`. The two functions have different options for ScrollBehavior. (And thanks to @patik - his work on #46654 saved me time in creating this issue!)

**Comment by BayanBennett** [comment]
Note that `behavior: "auto"` inherits the behavior from the CSS [scroll-behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior), which is why having the "instant" value is important

**Comment by BayanBennett** [comment]
Seems like there's a PR for this:
- https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1195

**Comment by BayanBennett** [comment]
More context: It seems like the "instant" value isn't part of the spec https://github.com/w3c/csswg-drafts/blob/main/cssom-view-1/Overview.bs#L407

**Comment by zhangenming** [comment]
![8b07e5831d8c3695a2fddbac139c908](https://user-images.githubusercontent.com/21235555/221778802-a44db562-3c43-4911-a795-09ecaae3657f.png)

auto and instant behave differently in my browser

**Comment by YahangWu** [comment]
This has been restored in cssom https://github.com/w3c/csswg-drafts/pull/8107, please added it back.

**Comment by curtisburns** [comment]
Just encountered this issue. Would be great to have this as MDN docs seem to suggest it is an option, and I only get the behaviour I need with `instant` and not `auto` which seems to fall back on the CSS style rule if set (I have it set to `smooth`). Have used `'instant' as ScrollBehavior` in the meantime. Cheers.

**Comment by Teamop** [comment]
it's already fixed in https://github.com/microsoft/TypeScript/pull/53133 which was just include in [v5.1-beta](https://github.com/microsoft/TypeScript/releases/tag/v5.1-beta)

**Comment by cbroomemktw** [comment]
@Teamop thanks! For anyone who can't update while still in beta you can brute force this setting as a workaround: 

```
      const scrollToOptions = { top: 0, left: 0, behavior: 'instant' };
      window.scrollTo(scrollToOptions as unknown as ScrollToOptions);
```

**Comment by DanielRuf** [comment]
> Note that `behavior: "auto"` inherits the behavior from the CSS [scroll-behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior), which is why having the "instant" value is important

Exactly.

**Comment by jakebailey** [comment]
As noted in https://github.com/microsoft/TypeScript/issues/47441#issuecomment-1514519335, this is already fixed and now released in TS 5.1.

**Comment by helloint** [comment]
> As noted in [#47441 (comment)](https://github.com/microsoft/TypeScript/issues/47441#issuecomment-1514519335), this is already fixed and now released in TS 5.1.

Can this small issue being published to latest 4.x? Upgrading to 5.x is a big change to the project and the entire eco
